app/views/state_registrants/pa/_step_3_newui2020.html.haml

```diff
@@ -0,0 +1,114 @@
+= render 'shared/signature_capture/scripts'
+- content_for :head do
+  :javascript
+    $(document).ready(function() {
+      function togglePenndotNumber() {
+        if ($("#state_registrants_pa_registrant_confirm_no_penndot_number:checked").length > 0) {
+          $("#penndot_set").hide();
+          // if (!window.IS_TOUCH_ENABLED) {
+          //   $("#signature_upload").show();
+          // }
+        } else {
+          $("#penndot_set").show();
+          
+          // $("#signature_upload").hide();
+        }              
+      }
+
+      function toggleSsn4() {
```

Comment:
can we remove togglePenndotNumber and toggleSsn4 and toggleAssistant if they're covered by the yes/no interface now?

Reply:
Yes.

Reply:
We could do with a Yes/ No "I do not know my SSN"  and a defaulting of that value to "No" so the form shows initially, and his hidden when they say "Yes"  -- I do not know my SSN.

Let me know on that.

Reply:
Actually.... Just checked Figma, and they explicitly left this as is with the checkbox after the fact.  So, I will leave as is.